Source/WTF/Configurations/Base.xcconfig

```diff
@@ -67,7 +67,7 @@ GCC_OBJC_CALL_CXX_CDTORS = YES;
 GCC_PRECOMPILE_PREFIX_HEADER = YES;
 GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) $(inherited);
 GCC_STRICT_ALIASING = YES;
-GCC_THREADSAFE_STATICS = NO;
+GCC_THREADSAFE_STATICS = YES;
```

Comment:
It would be slightly nicer to set this in Configurations/CommonBase.xcconfig and then un-set it in JavaScriptCore/Configurations/Base.xcconfig and bmalloc/Configurations/Base.xcconfig.